core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -764,8 +764,10 @@ private[spark] class Master(
       val replayBus = new ReplayListenerBus()
       val ui = SparkUI.createHistoryUI(new SparkConf, replayBus, new SecurityManager(conf),
         appName + status, HistoryServer.UI_PATH_PREFIX + s"/${app.id}")
+      val maybeTruncated = eventLogFile.endsWith(EventLoggingListener.IN_PROGRESS)
       try {
-        replayBus.replay(logInput, eventLogFile)
+        replayBus.replay(logInput, eventLogFile, 
```

Comment:
OK by me except this line no longer needs to wrap. Maybe see if others have a comment before bothering to update again.